<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerOutputBottomModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerOutputBottomModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_Output" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>491</width>
     <height>1033</height>
    </rect>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>441</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget::item
{
    padding-left:40px;
    padding-right:40px;
}</string>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>950</y>
      <width>208</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存并退出分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_preview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>950</y>
      <width>208</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt; 上一步 手术计划</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
